htdocs/libraries/icms/core/Session.php

```diff
@@ -44,25 +45,23 @@
  * Written by Vagharshak Tozalakyan <vagh@armdex.com>
  * Released under GNU Public License
  *
- * @category	ICMS
- * @package		Session
+ * @category ICMS
+ * @package Session
  *
- * @author	    Kazumi Ono	<onokazu@xoops.org>
- * @copyright	copyright (c) 2000-2003 XOOPS.org
+ * @author Kazumi Ono <onokazu@xoops.org>
+ * @copyright copyright (c) 2000-2003 XOOPS.org
  */
 class icms_core_Session {
 
 	/**
 	 * Initialize the session service
+	 *
 	 * @return icms_core_Session
 	 */
 	static public function service() {
 		global $icmsConfig;
 		$instance = new icms_core_Session(icms::$xoopsDB);
-		session_set_save_handler(
-			array($instance, 'open'), array($instance, 'close'), array($instance, 'read'),
-			array($instance, 'write'), array($instance, 'destroy'), array($instance, 'gc')
-		);
+		session_set_save_handler(array($instance, 'open'), array($instance, 'close'), array($instance, 'read'), array($instance, 'write'), array($instance, 'destroy'), array($instance, 'gc'));
```

Comment:
can you revert this to previous form? now it's harder to read - a bit too long

Reply:
Changed it back - and working on the code formatter